.ci/scripts/coverity.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -eEl
+
+topdir=$(git rev-parse --show-toplevel)
+cd $topdir
+
+
+if [ ! -d .git ]; then
+	echo "Error: should be run from project root"
+	exit 1
+fi
+
+echo "==== Running coverity ===="
+
+ncpus=$(cat /proc/cpuinfo|grep processor|wc -l)
+export AUTOMAKE_JOBS=$ncpus
+
+./autogen.sh
```

Comment:
One would expect UCC built before any testing.
Do we really have to build it again?